### PR TITLE
add parent id parameter to create command and enable user to bind script with any supported  document

### PR DESCRIPTION
**Pull request by bladechild**
add parent id parameter to "create" command and enable user to bind script with any supported  document

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -354,20 +354,21 @@ program
...
-  .action((title = LOG.UNTITLED_SCRIPT_TITLE) => {
+  .action((title = LOG.UNTITLED_SCRIPT_TITLE,parentId) => {
     if (fs.existsSync(DOT.PROJECT.PATH)) {
       logError(null, ERROR.FOLDER_EXISTS);
     } else {
       getAPICredentials(() => {
         spinner.setSpinnerTitle(LOG.CREATE_PROJECT_START(title));
         spinner.start();
-        script.projects.create({ title }, {}, (error, res) => {
+        script.projects.create({ title,parentId }, {}, (error, res) => {
```
Comment (grant) [inline index.js]: space after ,

File: index.js [hunk 2]
```diff
@@ -354,20 +354,21 @@ program
...
+ * Creates a new script project. The project title is optional. The project parent Id is optional.
+ * ParentId: The Drive ID of a parent file that the created script project is bound to. This is usually the ID of a Google Doc, Google Sheet, Google Form, or Google Slides file. If not set, a standalone script project is created.
  * @example `create "My Script"`
  */
 program
-  .command('create [scriptTitle]')
+  .command('create [scriptTitle] [scriptParentId]')
   .description('Create a script')
-  .action((title = LOG.UNTITLED_SCRIPT_TITLE) => {
+  .action((title = LOG.UNTITLED_SCRIPT_TITLE,parentId) => {
```
Comment (grant) [inline index.js]: space after ,

File: index.js [hunk 3]
```diff
@@ -354,20 +354,21 @@ program
   });
 
 /**
- * Creates a new script project. The project title is optional.
+ * Creates a new script project. The project title is optional. The project parent Id is optional.
+ * ParentId: The Drive ID of a parent file that the created script project is bound to. This is usually the ID of a Google Doc, Google Sheet, Google Form, or Google Slides file. If not set, a standalone script project is created.
```
Comment (grant) [inline index.js]: Please make line length <= 80.

File: index.js [hunk 4]
```diff
@@ -354,20 +354,21 @@ program
   });
 
 /**
- * Creates a new script project. The project title is optional.
+ * Creates a new script project. The project title is optional. The project parent Id is optional.
```
Comment (grant) [inline index.js]: Now that we have multiple parameters, can you describe them roughly like this?

```js
/**
 * Creates a new script project.
 * @param {string} [scriptTitle] An optional project title.
 * @param {string} [scriptParentId] An optional project parent Id. The Drive ID of a parent file
 *   that the created script project is bound to. This is usually the ID of a
 *   Google Doc, Google Sheet, Google Form, or Google Slides file. If not set, a
 *   standalone script project is created.
 * @see https://developers.google.com/apps-script/api/reference/rest/v1/projects/create#body.request_body.FIELDS.parent_id
 */
```

File: index.js [hunk 5]
```diff
@@ -354,20 +354,21 @@ program
...
+ * ParentId: The Drive ID of a parent file that the created script project is bound to. This is usually the ID of a Google Doc, Google Sheet, Google Form, or Google Slides file. If not set, a standalone script project is created.
  * @example `create "My Script"`
```
Comment (grant) [inline index.js]: Can you please add another line to include an example with a scriptParentId?

File: index.js (hunk 4 above)
Comment (bladechild) [reply index.js]: Sure, I will do it. Thanks

File: index.js (hunk 3 above)
Comment (bladechild) [reply index.js]: Thank you, I will do it

File: index.js (hunk 5 above)
Comment (bladechild) [reply index.js]: Yes, I will add an example soon

File: index.js (hunk 2 above)
Comment (bladechild) [reply index.js]: Thank you

File: index.js (hunk 1 above)
Comment (bladechild) [reply index.js]: Thank you

File: index.js [hunk 6]
```diff
@@ -323,7 +323,7 @@ program
 
       // Create a local HTTP server that reads the OAuth token
       var app = connect();
-      app.use(function(req, res) {
+      app.use(function (req, res) {
```
Comment (grant) [inline index.js]: Don't change this next time. (We should use an arrow function if we change).

Comment (bladechild) [reply index.js]: Thanks. The space after function is actually added by Eslint. I will pay more attention next time.

**Pull request merged**